admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -155,6 +155,17 @@ recommended if Redis is running on the same system as Nextcloud) use this exampl
 
 Only "host" and "port" variables are required, the other ones are optional.
 
+For the Redis side, you may need to add your web server user to the redis group::
+
+  usermod -a -G redis www-data
+
+If you were to use the Unix socket example right above, you'd edit and set the following lines in ``/etc/redis/redis.conf``::
+
+  port 0
+  unixsocket /var/run/redis/redis.sock
+  unixsocketperm 770
+  requirepass secret
```

Comment:
Do we want to set up a password here? Especially like this, people will just copy & paste and use the password `secret`. And don't you have to put this password in the config.php?

If you only run Nextcloud and don't use redis for anything else (as socket, you obviously can't use it via network).

Reply:
I would just go with a simple setup without password and via socket like in (from @jasonbayton):
https://bayton.org/docs/nextcloud/installing-nextcloud-on-ubuntu-16-04-lts-with-redis-apcu-ssl-apache/#6-2-enable-caching